readme.md

```diff
@@ -1,35 +1,45 @@
-json2xls
-========
+# json2xls
 
 [![Build Status](https://travis-ci.org/rikkertkoppes/json2xls.png?branch=master)](https://travis-ci.org/rikkertkoppes/json2xls)
 
-utility to convert json to a excel file, based on [Node-Excel-Export](https://github.com/functionscope/Node-Excel-Export)
+The utility for convert json to an excel file, based on [Node-Excel-Export](https://github.com/functionscope/Node-Excel-Export)

```

Comment:
The grammar here is not quite right I think. Other than that, the pr seems fine

Reply:
![image](https://user-images.githubusercontent.com/5930429/42128146-5ef69614-7cad-11e8-906f-d3fe9efcd82d.png)

Reply:
What exactly do you want me to do? @rikkertkoppes